examples/Sentinel_2_example_notebook/Sentinel2_example_notebook.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<blockquote> int to the </blockquote><p>with the?</p><p> </p><blockquote>previously defined,</blockquote><p>no comma</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>